tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py

```diff
@@ -7,16 +8,46 @@
 import threading
 import traceback
 
-
 from tendrl_gluster import TendrlGlusterfsMonitoringBase
 
+import utils as tendrl_glusterfs_utils
+
 
 class TendrlBrickUtilizationPlugin(
     TendrlGlusterfsMonitoringBase
 ):
+    etcd_client = {}
+
     def __init__(self):
         self.provisioner_only_plugin = False
         TendrlGlusterfsMonitoringBase.__init__(self)
+        if not self.etcd_client:
+            _etcd_args = dict(
+                host=self.CONFIG['etcd_host'],
```

Comment:
does the plugin config template have these `etcd_*` variables?

Reply:
yes, it has, i verified